src/views/create-dataset/CreateDatasetContainer.tsx

```diff
@@ -0,0 +1,77 @@
+// containers/FormContainer.tsx
+
+import React, { useState, ChangeEvent, FormEvent } from 'react'
+import { FormInputElement } from '@iqss/dataverse-design-system/src/lib/components/form/form-group/form-element/FormInput'
+import CreateDatasetUseCase from './CreateDatasetUseCase'
+import CreateDatasetFormPresenter from './CreateDataset'
+
+const CreateDatasetContainer: React.FC = () => {
+  const [formData, setFormData] = useState({
+    createDatasetTitle: ''
+  })
+
+  const [submitComplete, setSubmitComplete] = useState(false)
+
+  const [submitting, setSubmitting] = useState(false)
+
+  const handleCreateDatasetFieldChange = (event: ChangeEvent<FormInputElement>): void => {
+    const { name, value } = event.target
+    setFormData((prevData) => ({
+      ...prevData,
+      [name]: value
+    }))
+  }
+
+  const handleCreateDatasetSubmit = async (event: FormEvent<HTMLFormElement>): Promise<void> => {
```

Comment:
If you're following a Model - View - Presenter pattern, then I assume CreateDatasetContainer will be the view. Since the view is only is only responsible of the UI rendering in this MVP pattern, handling the inputs with `handleCreateDatasetFieldChange` and maintaining the form state `formData` should be handled by the Presenter